common/persistence/sql/sqlplugin/mysql/nexus_services.go

```diff
@@ -0,0 +1,77 @@
+// The MIT License
+//
+// Copyright (c) 2020 Temporal Technologies Inc.  All rights reserved.
+//
+// Copyright (c) 2020 Uber Technologies, Inc.
```

Comment:
I've been changing this to 2024 and not mentioning Uber in new files.

```suggestion
// Copyright (c) 2024 Temporal Technologies Inc.  All rights reserved.
```